scipy/linalg/tests/test_decomp.py

```diff
@@ -2502,5 +2503,39 @@ def test_orth():
         _check_orth(n)
 
 
+def test_subspace_angles():
+    H = hadamard(8, float)
+    A = H[:, :3]
+    B = H[:, 3:]
+    assert_allclose(subspace_angles(A, B), [np.pi / 2.] * 3, atol=1e-14)
+    assert_allclose(subspace_angles(B, A), [np.pi / 2.] * 3, atol=1e-14)
+    for x in (A, B):
+        assert_allclose(subspace_angles(x, x), np.zeros(x.shape[1]),
+                        atol=1e-14)
+    # From MATLAB
```

Comment:
It's just `subspace` called with the two matrices so probably not necessary to give the exact call (it depends on boilerplate loading data), but I should at least mention `subspace` since the name now diverges